PhysicsTools/PatUtils/interface/MiniIsolation.h

```diff
@@ -0,0 +1,31 @@
+#ifndef PhysicsTools_PatUtils_PFIsolation_h
+#define PhysicsTools_PatUtils_PFIsolation_h
+
+/*
+
+  Defines a MiniIsolation struct and a function to
+  compute MiniIsolation given a 4-vector and a collection
+  of packed PF candidates
+
+  Mini-Isolation reference: https://hypernews.cern.ch/HyperNews/CMS/get/susy/1991.html
+
+*/
+
+#include "DataFormats/Candidate/interface/Candidate.h"
+#include "DataFormats/PatCandidates/interface/PackedCandidate.h"
+#include "DataFormats/PatCandidates/interface/PFIsolation.h"
+
+namespace pat{
+
+    typedef ROOT::Math::LorentzVector<ROOT::Math::PxPyPzE4D<double> > LorentzVector;
```

Comment:
please use the typedef in DataFormats/Math/interface/LorentzVector.h
(this is a rather public namespace for a new generic typedef - and CMS has lots of different typedefs for LorentzVector...)